Sources/AppStoreConnectClient/AppStoreConnectClientProtocol.swift

```diff
@@ -24,19 +24,31 @@
 
 import Foundation
 
-/// A protocol defining the interface for searching the App Store Connect.
+/// A protocol defining the interface for interacting with the App Store Connect API.
```

Comment:
The protocol not equal implementation

Reply:
Changed to "A protocol that defines the interaction with the acquisition of programs, versions, builds, and previous versions".